lldb/tools/lldb-dap/OutputRedirector.cpp

```diff
@@ -25,6 +26,10 @@ using llvm::Expected;
 using llvm::inconvertibleErrorCode;
 using llvm::StringRef;
 
+namespace {
+char kCloseSentinel[] = "\0";
+} // namespace
```

Comment:
```suggestion
static constexpr auto kCloseSentinel[] = StringLiteral::withInnerNul("\0");
```

(removing the namespace per https://llvm.org/docs/CodingStandards.html#anonymous-namespaces, using a StringLiteral makes the rest of the code flow better)

Reply:
Done.